DuckDuckGoTests/TabViewControllerDaxDialogTests.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  TabViewControllerDaxDialogTests.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+import Persistence
+import Core
+@testable import DuckDuckGo
+
+final class TabViewControllerDaxDialogTests: XCTestCase {
+    private var sut: TabViewController!
+    private var delegateMock: MockTabDelegate!
+    private var onboardingPresenterMock: ContextualOnboardingPresenterMock!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        delegateMock = MockTabDelegate()
+        onboardingPresenterMock = ContextualOnboardingPresenterMock()
+        sut = .mock(contextualOnboardingPresenter: onboardingPresenterMock)
```

Comment:
I would not call this tab mock cause it’s a real tab I’d call it fixture? A standard initialisation

Reply:
Changed to "fake"